guide/16-geoenrichment-guides/part4_enrich_collections_and_sedf.ipynb

```diff
@@ -0,0 +1,2925 @@
+{
```

Comment:
<p>Is there an image supposed to be shown here? The preview is somehow missing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>It is supposed to be an iframe to show the data browser from here <a href="https://doc.arcgis.com/en/esri-demographics/data/data-browser.htm" target="_blank">https://doc.arcgis.com/en/esri-demographics/data/data-browser.htm</a>. Is it showing you the browser as in the link or is it just showing you the image you posted above?</p>